lib/webaction.js

```diff
@@ -116,6 +115,14 @@ async function invokeSelfAsync(params) {
 function webaction(main) {
     // actionWrapper -> webaction -> main (worker)
     return actionWrapper(async (params={}) => {
+        // Asset Compute Throttling Service metrics
+        // for quick feedback loop, store predicted run duration next to actual run duration
+        if (params && params.metrics && params.predictedRunDuration) {
+            console.log('params.metrics and params.predictedRunDuration are available');
+            params.metrics.add({
+                predictedRunDuration: params.predictedRunDuration
```

Comment:
qs, we use `estimatedRunDuration` for costTable metrics but `predictedRunDuration` here. do we want `estimatedRunDuration` to be 1:1? (this will require a small change in core mob-session)

Reply:
its the same as calling `estimatedRunDuration` here correct ? +1 on keeping the attribute name same at both places.

Reply:
+1 for same name

Reply:
ACK, will make the change

Reply:
just changed in core also: https://git.corp.adobe.com/nui/core/pull/326/commits/9dd71e5a8faf28d63f3ae23e380e636499a2d882